src/components/forms/select.scss

```diff
@@ -15,7 +15,8 @@
         padding-right: 4rem;
         width: 100%;
         height: 3rem;
-        text-indent: 1rem;
+        margin:0px;
```

Comment:
I understand why we want to remove text-indent, but please test on a few browsers and include screenshots, since I'm not sure why this was added in the first place!

But I'm not sure why we need, or want, `margin: 0`. That seems to slightly change the height of the whole Messages header bar. If I leave it out, the menu seems to look good...

Also:
* Put a space between the `:` and the `0`}
* change `0px` to just `0` (that's what we use in the other CSS, and IMHO it's a better practice)